starlite/params.py

```diff
@@ -115,7 +116,8 @@ def Body(
 
 
 @validate_arguments(config={"arbitrary_types_allowed": True})
-def Dependency(*, default: Any = Undefined) -> Any:
+def Dependency(*, default: Any = Undefined, skip_validation: bool = False) -> Any:
     """Creates a pydantic FieldInfo instance with an extra kwargs, used for
     both parameter parsing and OpenAPI schema generation."""
-    return Field(default, is_dependency=True)
+    extra: Dict[str, Any] = {EXTRA_KEY_IS_DEPENDENCY: True, EXTRA_KEY_SKIP_VALIDATION: skip_validation}
```

Comment:
I think this declaration is redundant, you can golf it into the function call

Reply:
Without it I get:

```
starlite/params.py:123: error: Argument 2 to "Field" has incompatible type "**Dict[str, bool]"; expected "Optional[Callable[[], Any]]"  [arg-type]
starlite/params.py:123: error: Argument 2 to "Field" has incompatible type "**Dict[str, bool]"; expected "str"  [arg-type]
```

Which I assume is why you did the same thing in `Path` and `Parameter`.